htdocs/includes/stripe/stripe-php/lib/Collection.php

```diff
@@ -2,6 +2,8 @@
 
 namespace Stripe;
 
+use Traversable;
```

Comment:
The includes dir is reserved for external lib.
We do not update files of external dir. Instead we must update the fill lib files with a new version provided by the maintener of the app.  Upgrade of external lib must be into a separated PR. 
Can you repush the PR without the changes into Stripe ?

Reply:
The stripe modification has been removed